src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -855,6 +859,211 @@ class TextimageOnOptionSegment(BaseSegment):
     )
 
 
+class TableOptionSegement(BaseSegment):
+    """TABLE option in `CREATE TABLE` statement.
+
+    https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql?view=sql-server-ver15
+    """
+
+    _ledger_view_option = Delimited(
+        Sequence(
+            "TRANSACTION_ID_COLUMN_NAME",
```

Comment:
These 4 are the same, can you bring them all into the same sequence, and wrap the keywords in a OneOf()?